crates/bevy_reflect/Cargo.toml

```diff
@@ -10,7 +10,9 @@ keywords = ["bevy"]
 readme = "README.md"
 
 [features]
+default = ["bevy"]
 bevy = ["glam", "smallvec", "bevy_math"]
+documentation = ["bevy_reflect_derive/documentation"]
```

Comment:
Does this feature need to be propagated up the crate hierarchy? Should we add it as a feature to `bevy_internal` and `bevy`?

Reply:
Nope, this is fine to be a leaf-crate feature.